betty/extension/nginx/serve.py

```diff
@@ -5,23 +5,23 @@
 from pathlib import Path
 from typing import Any
 
-import dill
 import docker
 from aiofiles.tempfile import TemporaryDirectory, AiofilesContextManagerTempDir
 from docker.errors import DockerException
 
 from betty.app import App
 from betty.extension.nginx.artifact import generate_dockerfile_file, generate_configuration_file
 from betty.extension.nginx.docker import Container
+from betty.project import Project
 from betty.serve import NoPublicUrlBecauseServerNotStartedError, AppServer
 
 
 class DockerizedNginxServer(AppServer):
     def __init__(self, app: App) -> None:
-        super().__init__(
-            # Create a new app so we can modify it later.
-            dill.loads(dill.dumps(app))
-        )
+        # Create a new project so we can modify it later.
+        project = Project()
```

Comment:
This requires the existing ancestry as well

Reply:
Also update the comment so it's clear we are doing this to update the project **configuration**